doc/connecting-to-data/customizing-data-appearance/feature-info-template.md

```diff
@@ -124,6 +124,21 @@ As with number you can also use `terria.dateTimeformat` directly in the template
 
               "featureInfoTemplate": "template": "{{#terria.formatDateTime}}{format: \"dd-mm-yyyy HH:MM:ss\"}2017-11-23T08:47:53Z{{/terria.formatDateTime}}</b>."
 
+## Replace text
+You can replace text by directly using `terria.replaceText` in the template and providing two arrays for matching and replacement. For example, with the following template,
+ - If the value of `feature.data.layerId` matches an element (number treated as string) in the `from` array, it will be replaced by corresponding string in the `to` array.
+ - If there is no matching in the `from` array, the value will not be replaced. 
+ - The flag `replaceText` must be defined and set to `true`. Otherwise no replacement will happen.
+ - Any unsafe string in the `to` array will be stripped off when being rendered.
+
+            "featureInfoTemplate": {
+              "template": '{{Pixel Value}} dwellings in {{#terria.replaceText}}{replaceText: true, from: [0,1,2,3], to: ["100m", "500m", "1km", "2km"]}{{feature.data.layerId}}{{/terria.replaceText}} radius.'
+            }
+
```

Comment:
Hey @mike, this looks good but I was thinking of a slightly different approach that could be a more generic solution. Mustache doesn't support rendering a partial using a variable name. But how about we add a function to do that? Then we can write this template like:

```json
"featureInfoTemplate": {
        "template": "{{Pixel Value}} dwellings in {{#terria.partialByName}}radius_{{feature.data.layerId}}{{/terria.partialByName}} radius.",
        "partials": {
          "radius_0": "100m",
          "radius_1": "500m",
          "radius_2": "1km",
          "radius_3": "2km"
        }
      }
```
We already support `partials` but the function will let the user switch partials depending on a property value.

This code change to do this will be something like in [this diff](https://gist.github.com/na9da/e771b28e71e571a107f36f65c136de53).